### PR TITLE
feat(doc-site): Adding doc-site

### PR DESCRIPTION
**Pull request by mnischay**
## Overview
 This is part 1 of the doc-site update [CR](https://github.com/awslabs/iot-app-kit/pull/778), which was having numerous test/build issues.

this only consists of the testing-util changes

UPDATE : merged the doc-site changes to this CR

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

**Review by corteggiano — APPROVED** [review]

**Comment by tracy-french** [comment]
@mnischay What's the plan for this PR?

**Comment by mnischay** [comment]
> @mnischay What's the plan for this PR?

good to merge, after Luci's approval, had a merge conflict

**Review by tracy-french — APPROVED** [review]

**Pull request merged**
